src/controllers/revocation.ts

```diff
@@ -846,14 +846,21 @@ export class RevocationController {
 				} as UpdateUnencryptedStatusListUnsuccessfulResponseBody);
 			}
 
+			// get status list
+			const unencrypted = await identityServiceStrategySetup.agent.searchStatusList2021(
+				did,
+				statusListName,
+				DefaultStatusActionPurposeMap[statusAction]
+			);
+
```

Comment:
This is redundant / repetitive as well.

You'll find what you're looking for at https://github.com/cheqd/credential-service/blob/34bf0cb607a81913d3e573de434f318ae8b6aca1/src/controllers/revocation.ts#L772.

Reply:
Done! ✅